src/libxtp/cartesiancoords.cc

```diff
@@ -0,0 +1,19 @@
+#include <string>
+#include <votca/xtp/cartesiancoords.h>
```

Comment:
Prefer quotation marks for internal headers over external library brackets. 

> Using quotes ("blah") instead of angle-brackets (<blah>) changes the compiler's behavior while it searches for the #included file.

> Using quotes makes the compiler look in the source file's local directory, then the default include directory. Using angle-brackets makes the compiler go directly to the default include directory and ignore the local directory when it searches.

> Basically, if you're writing a custom header, you should use quotes (i.e. "myClass.h") and use brackets for any standard headers (i.e. <iostream>, <cstdlib>, etc.).